### PR TITLE
docs: add style guide

### PR DESCRIPTION
**Pull request by themighty1**
This PR adds a minimal style guide.

### CODE REVIEW

**Review by yuroitaki — APPROVED** [review]
Looks good! @heeckhau should we also update the content of this file if we're not using the `CHANGELOG.md`
image

**Review by th4s — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -33,4 +33,35 @@ depending on the types of changes defined by
...
+## Style
+
+### Capitalization and punctuation
+
+Both line comments and doc comments must be capitalized. Each sentence must end with a period.
+
+```
+// This is a line comment.
+```
+
```
Comment (heeckhau) [inline CONTRIBUTING.md:47]: Are we sure about this?
I think enforcing the use of period makes sense for rustdoc comments. But it feels excessive for regular code comments.

**Review by heeckhau — APPROVED** [review]
Looks good. Just one remark

**Comment by heeckhau** [comment]
> Looks good! @heeckhau should we also update the content of this file if we're not using the `CHANGELOG.md`
I would postpone this until we switch to effectively using the conventional commit tooling.

**Pull request merged**